src/app/admin/pontuacao/edit-pontuacao/edit-pontuacao.component.ts

```diff
@@ -15,11 +19,15 @@ export class EditPontuacaoComponent implements OnInit {
   formPontuacao: FormGroup;
   pontuacaoObs: Observable<any>;
   querying: boolean = false;
+  delete;
+  update;
```

Comment:
ai, fazendo a modificação anterior você pode fazer algo como 
```suggestion
  update: any = [];
```